tests/5.0/requires/test_requires_unified_shared_memory_omp_target_alloc.c

```diff
@@ -0,0 +1,78 @@
+//===---test_requires_unified_shared_memory_omp_target_alloc.c --------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+// 
+// This test Checks for unified shared memory of an array that is allocated on 
+// the device and accessed in both host and device
+//
+// We use default mapping of pointers, which should be mappend as a zero lenght
+// array
+//
+////===----------------------------------------------------------------------===//
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 1024
+
+#pragma omp requires unified_shared_memory
+
+int unified_shared_memory_target_alloc() {
+  OMPVV_INFOMSG("Unified shared memory testing - Memory allocated in the device");
+  int errors = 0;
+  
+  int *anArray = (int *) omp_target_alloc(sizeof(int)*N, omp_get_default_device());
+  int anArrayCopy[N];
+
+  // Initialization
+#pragma omp target 
```

Comment:
I think we need to use is_device_ptr to use the existing device pointer.

Reply:
After reviewing during meeting, this should be correct according to the specs